src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.card {
+  font-size: 11px;
+  font-weight: 400;
+  font-family: $roboto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $card-width;
+  padding: 20px 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-bottom: 50px;
```

Comment:
block can't have margins, add mix class for card and move margin there
https://www.youtube.com/watch?v=JOJ-mENrg_s